expected-consensus.md

```diff
@@ -1,63 +1,64 @@
 # Expected Consensus
 
-This spec describes how to implement the protocol in general, for Filecoin-specific processes, see:
+This spec describes how the expected consensus (EC) protocol works in general. To read more about Filecoin-specific processes, see:
 
-- [Mining Blocks](mining.md#mining-blocks) on how consensus is used.
+- [Mining blocks](mining.md#mining-blocks) on how consensus is used in block mining.
 - [Faults](./faults.md) on slashing.
-- [Storage Market](./storage-market.md#the-power-table) on how the power table is maintained.
-- [Block data structure](data-structures.md#block) for details on fields and encoding.
+- [Storage market](./storage-market.md#the-power-table) on how the power table is maintained.
+- [Block data structure](data-structures.md#block) for details on fields and encoding
```

Comment:
are these cross references actually useful, one can just look at the table of contents

Reply:
Yeah, I think they're useful so people know where exactly expected consensus can be seen in the protocol